src/Microsoft.ServiceFabric.Services.Remoting/Microsoft.ServiceFabric.Services.Remoting.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.ServiceFabric.FabricTransport.Internal" Version="$(NugetPkg_Version_Microsoft_ServiceFabric_FabricTransport_Internal)" />
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="4.3.0" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="8.0.1" />
```

Comment:
@olegsych  - Do you think we need to release note these changes? Imagine user code as well taking dependency on these libraries and their code now throws compiler warnings, if the bindings redirects is not inclusive of the version we moved to.

Reply:
We don't have many packages and don't change them often, so it's not a bad thing to call out. I'm not sure how other package authors communicate these changes though. I imagine that teams that ship dozens of packages frequently would want to just point users to the dependencies listed on NuGet.org.

Reply:
I'll add release notes to the ADO work item.